booking.py

```diff
@@ -233,6 +273,22 @@ def add_tagged_friends(friends):
                 friends.append(Ntuple(friend_path, tdf_tree_friend))
         if not is_empty_file(root_file, tdf_tree):
             ntuples.append(Ntuple(root_file, tdf_tree, add_tagged_friends(friends)))
+            if validate_samples:
+                check_validity(root_file, validation_dict, friend_paths)
+    if len(validation_dict["errors"]) != 0:
+        logger.fatal("Validation for {} - {} - {} failed, differences were found".format(era, channel, dataset_name))
```

Comment:
Does the code below this line run? I am not sure if `logger.fatal` raises an error anymore.

Reply:
Logger Fatal just results in something like this:
```
ntuple_processor.booking - CRITICAL - Validation for 2018 - mt - qqH failed, differences were found
ntuple_processor.booking - CRITICAL - File /ceph/sbrommer/smhtt_ul/2022_07_v6/ntuples/2018/ZHToTauTau_M125_CP5_13TeV-powheg-pythia8_RunIISummer20UL18NanoAODv9-106X/mt/ZHToTauTau_M125_CP5_13T
eV-powheg-pythia8_RunIISummer20UL18NanoAODv9-106X_5.root has the following differences:
ntuple_processor.booking - CRITICAL -   {'THU_qqH_Mjj1000', 'THU_qqH_Mjj120', 'THU_qqH_TOT', 'THU_qqH_Mjj60', 'THU_qqH_Mjj350', 'THU_qqH_PTH200', 'THU_qqH_Mjj1500', 'THU_qqH_JET01', 'THU_qqH
_Mjj700', 'THU_qqH_25'}
```
but still continues running